.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+<!-- Thank you for working on Jamulus and opening a Pull Request! Please fill the following to make maintenance and development process visible -->
```

Comment:
Not exactly sure I understand what this is meant to say. Maybe something like this?

```suggestion
<!-- Thank you for working on Jamulus and opening a Pull Request! Please fill the following to make the review process straight foward -->
```

Reply:
It's not only about the review process, but also about the process of development (if there's any on the PR)